packages/vaadin-template-renderer/src/vaadin-template-renderer-templatizer.js

```diff
@@ -0,0 +1,63 @@
+import { PolymerElement } from '@polymer/polymer';
+import { templatize } from '@polymer/polymer/lib/utils/templatize';
+
+export class Templatizer extends PolymerElement {
+  static create(template) {
+    const templatizer = new Templatizer();
+    templatizer.__template = template;
+    return templatizer;
+  }
+
+  static get is() {
+    return 'vaadin-template-renderer-templatizer';
+  }
+
+  static get properties() {
+    return {
+      __template: Object,
+      __TemplateClass: Object,
+      __templateInstance: Object
+    };
+  }
+
+  /**
+   * @public
+   */
+  render(element) {
+    // TODO: Support creating multiple instances of the template
```

Comment:
The intention is to support creating multiple instances only when the need arises.